### DISCUSSION TITLE
Changing the admin panel path to empty string error

### DISCUSSION THREAD

**Discussion by dev-federico** [filamentphp/filament#7608] [Help]
As suggested in the documentation, I changed the admin panel path to an empty string. Unfortunately, I'm getting **Route [filament.admin.auth.login] not defined.** error. What could be the problem here? Thank you in advance!

**Comment by leandrocfe** [comment]
V3? did you set this?
```php
->login()
```
can you share the AdminPanelProvider.php ?

**Reply by dev-federico** [reply]
Hi, yes I'm using V.3 and I have set the "login()".
I solved this problem by adding also "domain()".
This is what I changed in the default file.

  public function panel(Panel $panel): Panel
    {
        return $panel
            ->id('admin')
            **->path('')**
            **->domain(env('APP_URL'))**
            ->login()
    ...

**Reply by danharrin** [reply]
Do you already have a `/login` route in your app

**Reply by dev-federico** [reply]
I think the problem can be related to the presence of Jetstream login page

**Reply by danharrin** [reply]
Yup

**Reply by dev-federico** [reply]
Thank you guys for your support! I will remove it
